views/templates/hook/displayBlock.tpl

```diff
@@ -44,8 +44,8 @@
                     {assign var=sizeCol_sm value=(12 / $numColsRemaining_md)}
                 {/if}
             {/if}
-            <div class="col-md-{$sizeCol_md} {$offsetCol_md} col-sm-{$sizeCol_sm} {$offsetCol_sm} col-xs-12"
-                {if $block['type_link'] !== $LINK_TYPE_NONE && !empty($block['link'])} style="cursor:pointer;" onclick="window.open('{$block['link']}')"{/if}>
+            <div class="col-md-{$sizeCol_md} {$offsetCol_md} col-sm-{$sizeCol_sm} {$offsetCol_sm} col-xs-12">
+                {if $block['type_link'] !== $LINK_TYPE_NONE && !empty($block['link'])} <a href="{$block['link']}" target="_blank"> {/if}
```

Comment:
from a UX point of view, I'm not fan of "target blank" attribute.

Reply:
@nicosomb I'm a fan of nearly everything (harmless, of course :) I chose `target="_blank"` to maintain `window.open` behavior. What do you suggest ?

Reply:
I think @leemyongpakvn has a point 😄

Reply:
@matks I'm going around the Earth neutrally so I have at least two points ;)